apps/recnet-api/src/database/repository/user.repository.type.ts

```diff
@@ -13,3 +13,20 @@ const userPreviews = Prisma.validator<Prisma.UserDefaultArgs>()({
 });
 
 export type UserPreview = Prisma.UserGetPayload<typeof userPreviews>;
+
+const user = Prisma.validator<Prisma.UserDefaultArgs>()({
```

Comment:
I think the `userPreviews` above should be change to `userPreview` for better consistency.